### PR TITLE
Multi file upload

### PR DESCRIPTION
**Pull request by sukrit-ym**

### CODE REVIEW

File: YMChat/src/main/java/com/yellowmessenger/ymchat/BotWebView.java [hunk 1]
```diff
@@ -259,6 +315,8 @@ public void onCreate(Bundle savedInstanceState) {
         setCloseButtonColorFromHex();
         setKeyboardListener();
 
+        WebView.setWebContentsDebuggingEnabled(true);
```
Comment (PurushYM) [inline YMChat/src/main/java/com/yellowmessenger/ymchat/BotWebView.java:318]: Please remove this debugging flag

File: app/src/main/java/com/yellowmessenger/ymchatexample/MainActivity.java [hunk 2]
```diff
@@ -65,6 +65,8 @@ protected void onCreate(Bundle savedInstanceState) {
         // This is an optional parameter
         ymChat.config.customLoaderUrl = "https://yellow.ai/images/Logo.svg";
 
+       // ymChat.config.customBaseUrl = "https://staging.yellowmessenger.com";
```
Comment (PurushYM) [inline app/src/main/java/com/yellowmessenger/ymchatexample/MainActivity.java:68]: Also remove this debugging code for staging

File: app/src/main/java/com/yellowmessenger/ymchatexample/MainActivity.java [hunk 3]
```diff
@@ -95,6 +97,7 @@ protected void onCreate(Bundle savedInstanceState) {
                     unlinkDevice();
                 }
         );
+        WebView.setWebContentsDebuggingEnabled(true);
```
Comment (PurushYM) [inline app/src/main/java/com/yellowmessenger/ymchatexample/MainActivity.java:100]: Remove this if not needed

**Review by PurushYM — APPROVED** [review]

File: YMChat/src/main/java/com/yellowmessenger/ymchat/BotWebView.java (hunk 1 above)
Comment (sukrit-ym) [reply YMChat/src/main/java/com/yellowmessenger/ymchat/BotWebView.java:318]: Will do

File: app/src/main/java/com/yellowmessenger/ymchatexample/MainActivity.java (hunk 2 above)
Comment (sukrit-ym) [reply app/src/main/java/com/yellowmessenger/ymchatexample/MainActivity.java:68]: sure

File: app/src/main/java/com/yellowmessenger/ymchatexample/MainActivity.java (hunk 3 above)
Comment (sukrit-ym) [reply app/src/main/java/com/yellowmessenger/ymchatexample/MainActivity.java:100]: Sure

**Pull request merged**